src/config/mod.rs

```diff
@@ -155,6 +155,8 @@ create_config! {
     emit_mode: EmitMode, EmitMode::Files, false,
         "What emit Mode to use when none is supplied";
     make_backup: bool, false, false, "Backup changed files";
+    inline_attribute_width: usize, 0, false,
+        "Attributes is in line when line width is within inline_attribute_width";
```

Comment:
Maybe it sounds clearer ?

```suggestion
        "Write an item and its attributes on the same line if their combine width is below a threshold";
```